docroot/profiles/custom/cgov_site/modules/custom/cgov_event/config/install/field.field.node.cgov_event.field_event_group.yml

```diff
@@ -0,0 +1,28 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_event_group
+    - node.type.cgov_event
+    - taxonomy.vocabulary.cgov_event_group
+id: node.cgov_event.field_event_group
+field_name: field_event_group
+entity_type: node
+bundle: cgov_event
+label: 'Event Group'
+description: 'Select the event group for which your event is associated.'
+required: true
+translatable: false
```

Comment:
this should be true per requirements

Reply:
This might have been on me to fix. I can't remember now. I just pushed a fix.